backend/src/main/java/de/bund/digitalservice/ris/norms/application/service/XsltTransformationService.java

```diff
@@ -39,10 +39,16 @@ public String transformLegalDocMlToHtml(TransformLegalDocMlToHtmlUseCase.Query q
       transformer.setOutputProperty(OutputKeys.ENCODING, "UTF-8");
       transformer.setParameter(SHOW_METADATA, query.showMetadata());
       transformer.setParameter("outputMode", "html");
-
+      String inputXml = query.xml();
+      if (query.snippet()) {
+        inputXml =
+            "<akn:akomaNtoso xmlns:akn=\"http://Inhaltsdaten.LegalDocML.de/1.6/\">"
```

Comment:
l like this, it also already solves the eventual problem that snippets do not include the namespace <3